plugins/resources/src/unixfilesystem.cpp

```diff
@@ -302,125 +279,113 @@ irods::error unix_file_mkdir_r(
 //      :: irods::error ret = _prop_map.get< double >( "my_key", my_var );
 // =-=-=-=-=-=-=-
 
-/// =-=-=-=-=-=-=-
 /// @brief interface to notify of a file registration
-irods::error unix_file_registered(
-    irods::plugin_context& _ctx ) {
-    irods::error result = SUCCESS();
-
+irods::error unix_file_registered(irods::plugin_context& _ctx)
+{
     // Check the operation parameters and update the physical path
-    irods::error ret = unix_check_params_and_path( _ctx );
-    result = ASSERT_PASS( ret, "Invalid parameters or physical path." );
+    if (const auto err = unix_check_params_and_path(_ctx); !err.ok()) {
+        return PASSMSG("Invalid parameters or physical path.", err);
+    }
 
     // NOOP
```

Comment:
How do we feel about potentially removing `unix_check_params_and_path` calls in the stub functions?

Reply:
what is the motivation behind removing them?

Reply:
yeah. if it wastes cycles and doesn't break anything then remove it.